src/Main.fs

```diff
@@ -36,9 +31,9 @@ let createTable() =
         |> unbox<IPerson array>
     div [] [
         hr []
-        p [] [str ("The text above has been parsed from markdown, " +
+        p [] [str ("The text2 above has been parsed from markdown, " +
```

Comment:
Please reverse this change I think you made it for debug purpose :)

Reply:
Yes, sorry. Tried to trigger the compiler.